### DISCUSSION TITLE
Netflix saving timestamps to kodi not on netflix?

### DISCUSSION THREAD

**Discussion by robbedoeske** [CastagnaIT/plugin.video.netflix#1041] [General]
When i watch a movie and i stop, normaly i can resume on another device. But with this addon this is not possible?

**Comment by ccope** [comment]
This was working at one point. If you're having problems, file an issue with debug logs attached.

**Reply by robbedoeske** [reply]
i mean on **another** device on same device is oke.

**Comment by CastagnaIT** [comment]
before full read Wiki meanings: https://github.com/CastagnaIT/plugin.video.netflix/wiki/Sync-of-watched-status-with-Netflix

**Reply by robbedoeske** [reply]
Stupid me,its the same as rtfm... Many thanks!
